pkg/util/pravegacluster.go

```diff
@@ -110,13 +110,13 @@ func LabelsForPravegaCluster(pravegaCluster *v1alpha1.PravegaCluster) map[string
 	}
 }
 
-func PvcIsOrphan(stsPvcName string, replicas int32) bool {
-	index := strings.LastIndexAny(stsPvcName, "-")
+func IsOrphan(stsName string, replicas int32) bool {
```

Comment:
We're not passing `stsName` here, as such this name is misleading.
Could we call this variable `k8sObjectName` ?

Reply:
done